action.yml

```diff
@@ -70,8 +73,22 @@ runs:
         service_account: ${{ inputs.service-account-email }}
         create_credentials_file: true
 
+    - name: 🪣 Cache firebase
+      uses: actions/cache@v4
+      with:
+        path: ./.firebase
+        key: ${{ runner.OS }}-firebase-${{ hashFiles('**/*.cache') }}
+        restore-keys: |
+          ${{ runner.OS }}-firebase-
+          ${{ runner.OS }}-
+
+    - name: ♻️ Set Firebase environment
+      if: inputs.environment != ''
+      shell: bash
+      run: npx firebase-tools use ${{ inputs.environment }}
```

Comment:
isn't this equivalent to using the --project flag like we use in the deploy and as an input? 

https://firebase.google.com/docs/cli/#:~:text=You%20can%20override,project%3Dprod.

Would we be able to update the value of `${{ inputs.project-id }}` to deploy to an environment?